### DISCUSSION TITLE
Firefox Browser Displays Blockeing Page When Accessing https://ci.bleachbit.org/

### DISCUSSION THREAD

**Discussion by justa-rowdy** [bleachbit/bleachbit#1692] [General]
This is more of an FYI only.  Got the following blocking page when accessing **https://ci.bleachbit.org/** using the Firefox Browser.

![image](https://github.com/bleachbit/bleachbit/assets/113783621/096fab54-e1eb-41eb-8738-6edec816367e)

Firefox is apparently using Google Safe Browsing who blocks this page likely due to Google's Unwanted Software Policy.

Not sure how long this block has been occurring, but you can easily get through it by clicking "See Details" and then, "ignore the risk" ,

**Comment by az0** [comment]
When I started upgrading BleachBit's framework from Python 3.4 to 3.10, Google would detect the Python 3.10  as harmful, and it affected Firefox, Google Chrome, and maybe other browsers. I read their documentation,  reported the issue, and wrote silly disclaimers on the ci.bleachbit.org pages, but the only reliable solution was to delete the Python 3.10 software.

Do you still see this? Lately I have not seen this issue, either with Python 3.10 (earlier) or Python 3.11 (now). That's important because I'm hoping to do a stable BleachBit release to update Python 3.4 to 3.11.

**Comment by justa-rowdy** [comment]
I just accessed https://ci.bleachbit.org/ using Firefox. I no longer get blocked. It goes straight to the page without issue.

As far as I can tell, I have two versions of Python on my Windows 10 Pro 64 bit machine.

3.4.4 and 3.9.13

I have not seen the issue you are describing.

Reminder though, I'm not a developer. And only used the BleachBit CI page once to fix a specific issue I had at the time and helped with some investigation/testing. 

I could try to update Python to the latest version for Windows, and see if anything changes?

**Reply by az0** [reply]
>I no longer get blocked. It goes straight to the page without issue

Great. I'm glad it works, so I will close this discussion.

>As far as I can tell, I have two versions of Python on my Windows 10 Pro 64 bit machine.
>I could try to update Python to the latest version for Windows, and see if anything changes?

Sorry, I was unclear. No, don't worry about it. BleachBit for Windows ships with a special build of Python, and for a while, Google was incorrectly flagging the BleachBit installer as harmful. Google was not specific, but it seemed to be related to the built-in Python version. In any case, it works now.
